AwesomeProject/res/colors.js

```diff
@@ -0,0 +1,11 @@
+export default {
+    blue: 'blue',
+    white: 'white',
+    dimgrey: 'dimgrey',
+    black: 'black',
+    red: 'red',
+    fonBlue: 'rgba(179, 217, 236, 0.8)',
+    text: 'rgba(64, 71, 75, 1)'
+}
+
+// export default { red: 'red', dimgrey: 'dimgrey'};
```

Comment:
delete unused code.

Reply:
you can merge this pull request after you fix this line. After this start task 2 :)